client/electron/main/launch-editor/index.ts

```diff
@@ -27,8 +27,14 @@ function isTerminalEditor(editor: string) {
   return false
 }
 
+function normalizePath(filename: string) {
+  const { win32, posix } = path
+  return filename.split(win32.sep).join(posix.sep)
```

Comment:
```suggestion
  return filename.replaceAll(win32.sep, posix.sep)
```

Would this be a bit more efficient?

Reply:
I think your consideration is right. I found this.
https://stackoverflow.com/questions/50463850/split-and-join-function-or-the-replace-function